examples/pxScene2d/src/rcvrcore/optimus.js

```diff
@@ -381,12 +446,21 @@ function Application(props) {
     _externalApp.on("onClientStopped", function () { _this.log( "onClientStopped"); }); // is never called
     _externalApp.ready.then(function() {
       _this.log("successfully created Spark app: " + _this.id);
-      _readyResolve();
+      _readyBaseResolve();
```

Comment:
Do we need to add a check here to confirm _readyBaseReject is valid before calling?  Can it still be undefined at this point?